### DISCUSSION TITLE
Best mid-range Duhua PTZ camera

### DISCUSSION THREAD

**Discussion by jakeycrx** [blakeblackshear/frigate#11908] [General]
Hi,

After reading the docs I can see Duhua is the camera to buy for PTZ, the problem is there are so many different models.  

What are peoples recommendations for a good mid-range PTZ camera that is proven to work in Frigate?

**Comment by NickM-27** [comment]
What features are you wanting? Support for Frigate Auto tracking or just ptz?

**Reply by jakeycrx** [reply]
Both if possible

**Reply by hawkeye217** [reply]
I developed autotracking for Frigate using a Dahua SD1A404XB-GNR mini PTZ, which is one of their lower end, smaller models. Any of the other larger ones should also have the necessary support.

**Reply by jakeycrx** [reply]
I forgot to mention POE would be preferable!

**Reply by hawkeye217** [reply]
The Dahua SD1A404XB-GNR is PoE, as are many of their mid and high end cameras.
